cypress/integration/login.js

```diff
@@ -22,4 +22,11 @@ describe('Login functionality', function() {
         cy.get('.user-info__title').contains('TestCon User 15')
         cy.get('.main-nav').find('li').should('have.length', 2)
     })
+    it('Check if date is selected as today', function () {
```

Comment:
for consistency purpose please leave a gap between test cases, or remove the gap between other test cases